app/models/user.rb

```diff
@@ -133,6 +118,10 @@ def handle_messages
     Message.where(sender: self, param_type: %w[group collection]).destroy_all
   end
 
+  def handle_user_identities
```

Comment:
Yes, I fully see your point. For now, we decided to go with this approach, since it is indeed in line with the existing code. Nevertheless, I second your opinion that the wording could be improved.